module/userProjects/helper.js

```diff
@@ -1594,6 +1594,14 @@ module.exports = class UserProjectsHelper {
                     
                     //compare & update userProfile with userRoleInformation
                     if ( projectCreation.data.userProfile && userRoleInformation && Object.keys(userRoleInformation).length > 0 && Object.keys(projectCreation.data.userProfile).length > 0 ) {
+
+                        if(!userRoleInformation){
```

Comment:
@praveenKDass Why check for !userRoleInformation inside when you are already checking userRoleInformation above on line 1596